gen/calc.py

```diff
@@ -70,6 +81,14 @@ def calculate_ip_detect_public_contents(ip_detect_contents):
     return ip_detect_contents
 
 
+def calculate_rexray_config_contents(rexray_config_filename):
+    try:
+        with open(rexray_config_filename, encoding='utf-8') as f:
+            return yaml.dump(f.read())
+    except IOError as err:
+        raise Exception('REX-Ray config file {}: {}'.format(rexray_config_filename, err))
```

Comment:
Why do we "rewrite" that exception here with information loss, instead of just letting this blow up (which yields the more complete log)?